src/styles/card.scss

```
@@ -0,0 +1,103 @@
+.card {
+  box-sizing: border-box;
+
+  width: 200px;
+  padding: 32px 16px 16px;
+
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+
+  background-color: white;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__img {
+    margin-bottom: 40px;
+    height: 134px;
+    width: 160px;
+  }
+
+  &__name {
+    margin-bottom: 4px;
+
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: $darkBlue;
+  }
+
+  &__code {
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    color: $lightGray;
+
+    width: 100%;
+    margin-bottom: 16px;
+  }
+
+  &__rate {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+
+    width: 100%;
+    margin-bottom: 24px;
+  }
+
+  &__rate-feedback {
+    font-weight: 400;
+    font-size: 10px;
+    color: $darkBlue;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+
+    width: 100%;
+    margin-bottom: 16px;
+  }
+
+  &__price-text {
+    font-weight: 400;
+    font-size: 12px;
+    line-height: 18px;
+    color: $lightGray;
+  }
+
+  &__price-value {
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 18px;
+    color: $darkBlue;
+  }
+
+  &__button {
+    text-decoration: none;
+
+    box-sizing: border-box;
+
+    width: 166px;
+    height: 40px;
+    background: #{$lightBlue};
+    border-radius: 5px;
+    border: 1px transparent solid;
+
+    line-height: 40px;
+    font-weight: 700;
+    font-size: 14px;
+    text-align: center;
+    text-transform: uppercase;
+    color: #fff;
+  }
+
+  &__button:hover {
+    color: #{$lightBlue};
+    background-color: white;
+
+    border: 1px solid #{$lightBlue};
```

you have a transparent border, you need just update the color
```suggestion
    border-color: #{$lightBlue};
```